src/services/formatting/formatting.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash';
```

Comment:
src/services/formatting look like a "fourre tout", may be organize thing by functionnal way.

src/i18n/formatting